public/js/save.js

```diff
@@ -86,7 +129,7 @@ $(document).ready(function() {
         for(var key in currentFormJS){
```

Comment:
Please check out https://developer.mozilla.org/en-US/docs/Web/API/FormData/entries for working with form data.